dashboard/app/controllers/api/v1/pd/regional_partner_workshops_controller.rb

```diff
@@ -18,26 +18,25 @@ def find
 
     # Find the matching partner, even if it has no workshops
     partner = @partners.find_by_region(zip_code, state) || RegionalPartner.find_by_region(zip_code, state)
-    render json: serialize_partner_workshops(partner)
+    if partner.nil?
+      # TODO: should we instead render json which has the keys but nil values, to preserve previous behavior?
+      render json: {}
+    else
+      render json: partner,
+             serializer: Api::V1::Pd::RegionalPartnerWorkshopsSerializer,
+             scope: {course: @course, subject: @subject}
+    end
   end
 
   # GET /api/v1/pd/regional_partner_workshops
   def index
-    render json: @partners.map {|p| serialize_partner_workshops(p)}
+    render json: @partners,
+           each_serializer: Api::V1::Pd::RegionalPartnerWorkshopsSerializer,
+           scope: {course: @course, subject: @subject}
   end
 
   private
 
-  def serialize_partner_workshops(partner)
-    # The scope is not being passed to the serializer with `render json: serializer:` syntax,
-    # so initialize this explicitly.
```

Comment:
confirming that this behavior has changed in the new version to allow the scope parameter with `render json: serializer:` syntax?

Reply:
Yep, it has.